Formula/z/zsh-completions.rb

```diff
@@ -38,6 +38,7 @@ def caveats
       Additionally, if you receive "zsh compinit: insecure directories" warnings when attempting
       to load these completions, you may need to run this:
 
+        chmod go-w '/opt/homebrew/share'
```

Comment:
doh, sorry, this probably should be using `#{HOMEBREW_PREFIX}` 🤦‍♂️ 

```suggestion
        chmod go-w '#{HOMEBREW_PREFIX}/share'
```